index.html

```diff
@@ -2,21 +2,130 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"
-    />
-    <title>My Blog</title>
-    <link
-      href="//fonts.googleapis.com/css?family=Roboto:400,500,300"
-      rel="stylesheet"
-      type="text/css"
-    />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <link rel="stylesheet" href="css/style.css" />
-    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <link rel="shortcut icon" type="image/x-icon" href="images/favicon.ico" />
+    <title>Nagas</title>
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <div id="app-container">
+      <!-- Semantically, I checked if <nav> MUST be within <header> but there is no hard rule -->
+      <!-- dev.w3.org/html5/spec/Overview.html#the-nav-element -->
+      <nav>
+        <h3 class="nav-title">Navigation</h3>
```

Comment:
Headings should be sequential in the document. This is one exception to the rule when you have headings before the main title of the page, in this circumstance I would start at h2